website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,682 @@
+                              className="w-full h-full object-contain"
+                              onError={e => {
+                                // Hide image if it fails to load
+                                const target = e.target as HTMLImageElement;
+                                target.style.display = 'none';
+                              }}
+                            />
+                          </div>
+                          <div className="min-w-0 flex-1">
+                            <h3
+                              onClick={() => handleExtensionClick(extension)}
+                              className="text-lg font-semibold text-charcoal-300 dark:text-white group-hover:text-purple-600 transition-colors truncate mb-2 cursor-pointer hover:text-purple-600 dark:hover:text-purple-400">
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)